warehouse/query-core/src/main/java/datawave/query/scheduler/PushdownScheduler.java

```diff
@@ -108,40 +108,23 @@ public void addSetting(IteratorSetting customSetting) {
             throw new IllegalArgumentException("Null configuration provided");
         }
         
-        try {
-            
-            return concatIterators();
-        } catch (AccumuloException | ParseException | TableNotFoundException | AccumuloSecurityException e) {
-            throw new RuntimeException(e);
-        }
-        
+        return concatIterators();
     }
     
     /**
      * @return
-     * @throws ParseException
-     * @throws TableNotFoundException
-     * @throws AccumuloSecurityException
-     * @throws AccumuloException
      */
-    protected Iterator<Entry<Key,Value>> concatIterators() throws AccumuloException, AccumuloSecurityException, TableNotFoundException, ParseException {
+    protected Iterator<Entry<Key,Value>> concatIterators() {
         
         String tableName = config.getShardTableName();
         
         Set<Authorizations> auths = config.getAuthorizations();
         
-        TabletLocator tl;
-        
         Instance instance = config.getConnector().getInstance();
         if (instance instanceof InMemoryInstance) {
-            tl = new InMemoryTabletLocator();
-            tableId = config.getTableName();
-        } else {
-            tableId = Tables.getTableId(instance, tableName);
-            Credentials credentials = new Credentials(config.getConnector().whoami(), new PasswordToken(config.getAccumuloPassword()));
-            tl = TabletLocator.getLocator(new ClientContext(instance, credentials, AccumuloConfiguration.getDefaultConfiguration()), tableId);
+            tableName = config.getTableName();
```

Comment:
I think this block can go away. As far as I can tell, `config.getTableName()` and `config.getShardTableName()` will return the same thing, so the set on line 129/119 should take care of it.